libs/liquidity/src/lib/liquidity-manager.tsx

```diff
@@ -0,0 +1,132 @@
+  partyId,
+  marketId,
+}: LiquidityManagerProps) => {
+  const gridRef = useRef<AgGridReact | null>(null);
+  const {
+    data: marketLiquidityData,
+    loading,
+    error,
+  } = useQuery<MarketLiquidity>(MARKET_LIQUIDITY_QUERY, {
+    variables: useMemo(() => ({ marketId }), [marketId]),
+  });
+
+  const { liquidityProviders, suppliedStake, targetStake, code, symbol } =
+    useLiquidityProvision({ data: marketLiquidityData });
+
+  const wrapperClasses = classNames(
+    'h-full max-h-full',
+    'flex flex-col',
+    'text-ui-small',
+    'gap-8'
+  );
+  const titleClasses =
+    'sm:text-lg md:text-xl lg:text-2xl flex items-center gap-4 whitespace-nowrap';
+
+  const myLpEdges = useMemo(
+    () => (liquidityProviders || []).filter((e) => e.party === partyId),
+    [liquidityProviders, partyId]
+  );
+  const activeEdges = useMemo(
+    () =>
+      (liquidityProviders || []).filter(
+        (e) => e.status === LiquidityProvisionStatus.STATUS_ACTIVE
+      ),
+    [liquidityProviders]
+  );
+  const inactiveEdges = useMemo(
+    () =>
+      (liquidityProviders || []).filter(
+        (e) => e.status !== LiquidityProvisionStatus.STATUS_ACTIVE
+      ),
+    [liquidityProviders]
+  );
+
+  const getActiveDefaultId = () => {
+    if (activeEdges?.length) return 'active';
+    else if (inactiveEdges?.length > 0) return 'inactive';
+    return 'active';
+  };
+  const itemClass =
+    'min-w-min w-[120px] whitespace-nowrap pb-3 px-4 border-l border-neutral-300 dark:border-neutral-700';
+  const itemHeading = 'text-neutral-400';
+  return (
+    <AsyncRenderer loading={loading} error={error} data={liquidityProviders}>
+      <header className="w-screen xl:px-4 pt-4 border-b border-neutral-300 dark:border-neutral-700">
+        <div className="xl:flex xl:gap-4  items-start">
+          <div className="px-4 mb-2 xl:mb-0">
+            <a className={titleClasses} href={`/markets/${marketId}`}>
+              {`${code} ${t('liquidity provision')}`}
+            </a>
+          </div>
+          <div className="flex flex-nowrap items-start xl:flex-1 w-full overflow-x-auto text-xs ">
+            <div className={itemClass}>
+              <div className={itemHeading}>{t('Target stake')}</div>
+              <div>{`${targetStake} ${symbol}`}</div>
+            </div>
+            <div className={itemClass}>
+              <div className={itemHeading}>{t('Supplied stake')}</div>
+              <div>{`${suppliedStake} ${symbol}`}</div>
+            </div>
+            <div className={classNames('col-span-2', itemClass)}>
+              <div className={itemHeading}>{t('Market ID')}</div>
+              <div style={{ wordBreak: 'break-word' }}>{marketId}</div>
+            </div>
+          </div>
+        </div>
+      </header>
+      <div className={wrapperClasses}>
+        <ResizableGrid vertical={true}>
+          {partyId && (
+            <ResizableGridPanel preferredSize={300} minSize={50}>
+              <Tabs activeDefaultId="myLP">
```

Comment:
chore (blocking): Why use Tabs component if there is only one tab? I feel like it might be because you want the tab title UI above the table. 

Lets keep it more simple and just render a My LP tab alongside active and inactive

Reply:
yeah, I am happy to do that. that's what I was proposing yesterday. check this out: 
<img width="1680" alt="Screenshot 2022-09-02 at 10 33 49" src="https://user-images.githubusercontent.com/16125548/188093584-fc05b1b8-a7f0-4879-84b5-06cb4629e927.png">
<img width="1680" alt="Screenshot 2022-09-01 at 23 46 54" src="https://user-images.githubusercontent.com/16125548/188093901-aa1deccb-358e-47de-bf8b-98426e76c404.png">
updated.